### ISSUE TITLE
Unable to remove all user-defined variables

### ISSUE THREAD

**Issue by Punyam-Singh** [spyder-ide/spyder#12963]
### What steps will reproduce the problem?

First I ran some python codes with a few variables.
Then i clicked the option of "remove all variables".
But the variables were not removed.
there was no change in reclicking the button too.
So i tried to manually remove the variables.
the first one tried was "DOR_Label" but it did not work and i faced the error.

### Traceback
```python-traceback
Exception in comms call remove_value:
  File "C:\Users\Ebird\Anaconda3\lib\site-packages\spyder_kernels\comms\commbase.py", line 342, in _handle_remote_call
    buffer['call_kwargs'])
  File "C:\Users\Ebird\Anaconda3\lib\site-packages\spyder_kernels\comms\commbase.py", line 353, in _remote_callback
    *call_args, **call_kwargs)
  File "C:\Users\Ebird\Anaconda3\lib\site-packages\spyder_kernels\console\kernel.py", line 236, in remove_value
    ns.pop(name)
KeyError: 'DOR_Label'
```

## Versions

* Spyder version: 4.1.3 
* Python version: 3.7.7
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 8.1

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.4.1 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.13.0 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.2 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=0.25                  :  2.5.0 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.10 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.0 (OK)
qtconsole >=4.6.0              :  4.7.3 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.0.3 (OK)
spyder_kernels >=1.9.1;<1.10.0 :  1.9.1 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  18.1.1 (OK)

# Optional:
cython >=0.21                  :  0.29.17 (OK)
matplotlib >=2.0.0             :  3.1.3 (OK)
numpy >=1.7                    :  1.18.1 (OK)
pandas >=0.13.1                :  1.0.3 (OK)
scipy >=0.17.0                 :  1.4.1 (OK)
sympy >=0.7.3                  :  1.5.1 (OK)
```

**Comment by goanpeca** [comment]
Hi @Punyam-Singh thanks for the report. I think this has been fixed and will be available in the next release.

---

@dalthviz I cannot recall if this has been fixed or if is this dependent on a newer version of spyder-kernels?

**Comment by dalthviz** [comment]
I fixed an issue where the variables were deleted individually (I think the fix should be available with Spyder 4.1.3 and Spyder kernels 1.9.1). I could take a look :+1: (probably is something similar to the individual variable elimination case but seems like the change didn't cover the case were the option to remove all the variables is used).

**Comment by ccordoba12** [comment]
> Then i clicked the option of "remove all variables".

Due to that, this issue is a duplicate of issue #12689. It'll be fixed in 4.1.4, to be released in a couple of weeks. Thanks for your patience @Punyam-Singh.
